src/utils/http/errors/FetchError.js

```diff
@@ -0,0 +1,12 @@
+class FetchError extends Error {
+  constructor(error) {
+    super(error);
+
+    Object.setPrototypeOf(this, new.target.prototype);
+    this.name = 'Fetch Error';
+    this.isOperational = true;
```

Comment:
What does `isOperational` mean?

Reply:
https://www.joyent.com/node-js/production/design/errors#:~:text=two%20broad%20categories%3A-,3,-Operational%20errors%20represent

I just added some parameters from examples I found, but it's not used.

Reply:
if its not used could we remove it?

Reply:
done